vips/vip-213/vip-213.ts

```diff
@@ -0,0 +1,63 @@
+import { ethers } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const XVS_BRIDGE_ADMIN = "0x70d644877b7b73800E9073BCFCE981eAaB6Dbc21";
+export const XVS_BRIDGE_SRC = "0xf8F46791E3dB29a029Ec6c9d946226f3c613e854";
+export const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+export const COMPTROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
+export const ETHEREUM_TREASURY = "0xFD9B071168bC27DBE16406eC3Aba050Ce8Eb22FA";
+export const XVS = "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63";
+
+export const RECEIVER_ADDRESS = ethers.utils.defaultAbiCoder.encode(["address"], [ETHEREUM_TREASURY]);
+export const ADAPTER_PARAMS = ethers.utils.solidityPack(["uint16", "uint256"], [1, 300000]);
+export const XVS_AMOUNT = parseUnits("75750", 18);
+export const DEST_CHAIN_ID = 101;
+
+export const vip213 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP to transfer XVS to ethereum chain",
+    description: ``,
+
+    forDescription: "I agree that Venus Protocol should proceed with this transfer for XVS",
+    againstDescription: "I do not think that Venus Protocol should proceed with this transfer for XVS",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds with this transfer for XVS",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: COMPTROLLER,
+        signature: "_grantXVS(address,uint256)",
+        params: [NORMAL_TIMELOCK, XVS_AMOUNT],
+      },
+      {
+        target: XVS,
+        signature: "approve(address,uint256)",
+        params: [XVS_BRIDGE_SRC, XVS_AMOUNT],
+      },
+      {
+        target: XVS_BRIDGE_SRC,
+        signature: "sendFrom(address,uint16,bytes32,uint256,(address,address,bytes))",
+        params: [
+          NORMAL_TIMELOCK,
+          DEST_CHAIN_ID,
+          RECEIVER_ADDRESS,
+          XVS_AMOUNT,
+          [NORMAL_TIMELOCK, ethers.constants.AddressZero, ADAPTER_PARAMS],
+        ],
+        value: "500000000000000000",
```

Comment:
The [VIP 271](https://github.com/VenusProtocol/vips/pull/228) is going to transfer the BNB in the Normal Timelock to the Venus Treasury, so this command would fail.

My suggestion: transfer 0.5 BNB from the Treasury to the Normal Timelock in a previous command, and set the address of the Venus Treasury as the receiver of the BNB not used in the bridge command

Reply:
[9e58806](https://github.com/VenusProtocol/vips/pull/119/commits/9e588062e0199dcdafd35343782685d3b80f7b7b)_